extensions/amp-a4a/0.1/test/test-real-time-config-manager.js

```diff
@@ -138,8 +154,10 @@ describes.realWin('real-time-config-manager', {amp: true}, env => {
       return urls;
     }
 
-    function rtcEntry(response, url, error) {
-      const callout = getCalloutParam(url);
+    function rtcEntry(response, url, error, isVendor) {
+      // If this is an entry for a vendor, then the callout is just
+      // the vendor name passed in to url here.
+      const callout = !!isVendor ? url : getCalloutParam_(url);
```

Comment:
minor: is that `!!` actually doing anything?

Reply:
I suppose not, I just put it there for sanity sometimes as I'm going